extension_helpers/_openmp_helpers.py

```diff
@@ -134,8 +135,41 @@ def get_openmp_flags():
             link_flags.append('-L' + lib_path)
             link_flags.append('-Wl,-rpath,' + lib_path)
 
-        compile_flags.append('-fopenmp')
-        link_flags.append('-fopenmp')
+        if not check_apple_clang():
+            compile_flags.append('-fopenmp')
+            link_flags.append('-fopenmp')
+        else:
+            msg = textwrap.dedent(
+                """\
+                You are using Apple Clang compiler.
+
+                Your system should be prepared:
+                1. You should have specfically installed OpenMP,
+                   for instance by running `brew install libomp`.
+                2. OpenMP source and library should be findable by the compiler.
+
+                By default, OpenMP source and library will be looked in
+                `/usr/local/opt/libomp/include` and `/usr/local/opt/libomp/lib`
+                respectively.
+
+                To use specific OpenMP source and library paths, you can setup
+                the following environment variables `CFLAGS` and `LDFLAGS`
+                before any compilation/installation, e.g.
+                ```
+                export CFLAGS="-I/usr/local/opt/libomp/include"
+                export LDFLAGS="-L/usr/local/opt/libomp/lib"
+                ```
+                """
+            )
+            log.warn(msg)
+            compile_flags.append('-Xpreprocessor -fopenmp')
+            if not 'CFLAG' in os.environ and \
```

Comment:
Should this be `CFLAGS` (and `LDFLAGS` below)?

Reply:
Yes, thanks! I fixed it in 094b820